src/site/_includes/layout.njk

```diff
@@ -165,6 +165,7 @@ show_banner: true
     </web-side-nav>
     <main>
       <div id="content">
+        <input class="hidden" id="w-toc__open-state" type="checkbox">
```

Comment:
Hm I think this'll be added to every page. Maybe we don't need this if we decide to use a real component that's only displayed on posts.